manifests/init.pp

```diff
@@ -129,6 +133,7 @@
   Boolean $service_enable              = $telegraf::params::service_enable,
   String  $service_ensure              = $telegraf::params::service_ensure,
   Array   $install_options             = $telegraf::params::install_options,
+  String  $merge_strategy         = $telegraf::params::merge_strategy,
```

Comment:
Since empty strings won't work here, we can enforce the minimal string length:
```suggestion
  String[1]  $merge_strategy         = $telegraf::params::merge_strategy,
```

even cooler would it be if we use an enum that enforces the different merge options. Can you please update the code and also align the `=`?